README.md

```diff
@@ -7,73 +7,82 @@
-- It allows you to start writing complex test cases in a minute.
-- It makes a lot of work for you: download and install Selenium server with one command; set-up browser of your choice; automatically take a screenshot on failed assertion; produce test results in JUnit format (easily processable e.g. by Jenkins and other tools) and more.
-- Your tests are run in a parallel, so the bottleneck is just the amount of Selenium nodes you start simultaneously.
-- Simple syntax sugar layer on top of default [WebDriver commands](https://github.com/facebook/php-webdriver/wiki/Example-command-reference) can help you shorten your tests and make them more readable.
-- If you already use PHP, you don't have to learn a new language to write functional tests. Moreover, if you are familiar with unit tests and PHPUnit, you know it all.
-- Allows you to plan tests dependencies - need to wait 2 minutes until some event gets through your message queue so you could test the result? No problem! The order of tests is even optimized to minimize the total execution time.
-- Status of the tests could be clearly watched during tests execution, so you will easily know how many tests were already finished and what was their result.
-- You can extend it easily by e.g. registering custom events to EventDispatcher. Thus you can, for example, add custom configuration options or change parameters passed to child PHPUnit processes.
-- Cloud services like [Sauce Labs](https://saucelabs.com/), [BrowserStack](https://www.browserstack.com/) or [TestingBot](https://testingbot.com/) are fully integrated, giving you a possibility to run tests with even less setup and without own infrastructure.
-- It is field tested - we use it daily in [our company](https://www.lmc.eu/english) to maintain quality of our various products thanks to hundreds of test-cases. The library itself is also extensively covered with unit tests.
+- It allows you to start writing complex test cases in a minute
+- It performs a lot of work for you:
+   - downloads and installs Selenium server with one command
+   - sets-up browser of your choice
+   - automatically takes a screenshot on failed assertions
+   - produces test results in JUnit format (easily processable e.g. by Jenkins and other tools)
+   - and more...
+- Tests are run in parallel, so the only bottleneck is the number of Selenium nodes you start simultaneously
+- Simple syntax sugar layer on top of default [WebDriver commands](https://github.com/facebook/php-webdriver/wiki/Example-command-reference) helps shorten your tests and improve readability
+- If you already use PHP, you don't have to learn a new language to write functional tests. Moreover, if you are familiar with unit tests and PHPUnit, you know it all
+- Allows you to plan test dependencies
+   - Ex. need to wait 2 minutes until some event gets through your message queue before testing the result? No problem! The order of tests is optimized to minimize the total execution time
```

Comment:
Eg? Or if this is shortcut for example, I'd prefer longer version without the shortcut ("For example ..."). Also the same case below.